### PR TITLE
Transport Permit form

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#18313

*Description of changes:*
- Transport Permit Form components for Staff only (QS is coming in next PR)
- Refactor few components to make them reusable for this flow
- Refactor Land or Lease component
- Styling updates/fixes
- Unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1689-ef68dsqp.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue [hunk 1]
```diff
@@ -218,10 +218,44 @@ import { HomeLocationTypes } from '@/enums'
...
+    section: '',
+    township: '',
+    range: '',
+    meridian: '',
+    landDistrict: '',
+    plan: '',
+    bandName: '',
+    reserveNumber: '',
+    exceptionPlan: ''
+}
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue]: If this isn't used anywhere but the prop default, could define in the prop.

File: ppr-ui/src/components/mhrTransfers/LocationChange.vue [hunk 2]
```diff
@@ -14,16 +14,70 @@
...
+        <p class="mt-2">
+          <b>Note:</b> If this manufactured home is being moved to a location outside of B.C.,
+          the status of the home will be exempt upon filing.
+        </p>
+
+        <HomeCivicAddress
+          :schema="CivicAddressSchema"
+          :validate="false"
+          :class="{ 'border-error-left': false }"
+          @isValid="() => {}"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/LocationChange.vue:60]: Is this for part 2?

**Review by cameron-eyds — APPROVED** [review]
LGTM!

File: ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue]: Yea this is just a for the prop, wanted to keep the component looking cleaner, but can certainly update it.

File: ppr-ui/src/components/mhrTransfers/LocationChange.vue (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/components/mhrTransfers/LocationChange.vue:60]: Yea, validations are in the other ticket.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1689-ef68dsqp.web.app

**Comment by dimak1** [comment]
@cameron-eyds I added a Tax Certificate component, if you want to take another look.

**Comment by cameron-eyds** [comment]
> @cameron-eyds I added a Tax Certificate component, if you want to take another look.

@dimak1 Looks great!

**Review by doug-lovett — APPROVED** [review]
Changes look great.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1689-ef68dsqp.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1689-ef68dsqp.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1689-ef68dsqp.web.app

**Pull request merged**